pages/dkp/kommander/2.2/install/small-environment/index.md

```diff
@@ -30,60 +32,69 @@ In this YAML file you can find commented lines, that correspond to all platform
 
 For example, if you want to test Grafana to allow monitoring, remove the pound sign preceding `grafana-logging: null`. Since Grafana depends on Loki, you also have to remove the pound preceding `grafana-loki: null` and `minio-operator: null`. Note that depending on the size of your cluster, enabling several platform applications could exhaust your cluster’s resources.
```

Comment:
the yaml below switches to the new `enabled` structure (which is supported in 2.2). If we want to change it, we'll need to update this paragraph here (and maybe other places?) to use it instead of `null`. can we also make the same changes that I made in the confluence docs, some adjustment to this paragraph and moving it above the "important" block?

```
For example, if you want to enable the logging stack, set `enabled` to `true` for `grafana-logging`, `grafana-loki`, `logging-operator` and `minio-operator`. Note that depending on the size of your cluster, enabling several platform applications could exhaust your cluster’s resources.
```

Reply:
Hi @gracedo!
I implemented those changes and added a few more files in the PR to update the new logic (going from `null` to `enabled: true` or `false`). Could you PTAL?